nupic/research/frameworks/vernon/parser_utils.py

```diff
@@ -175,6 +174,9 @@ def process_args(args, config):
         insert_experiment_mixin(config, ray_wandb.WorkerLogger, prepend_name=False)
 
     if "create_sigopt" in args:
+
+        from nupic.research.frameworks.sigopt import SigOptExperiment
```

Comment:
This is not related to your change, but this parsing of the args doesn't look right. You should be able to run SigOptExperiment without creating a new experiment (e.g. you just want to run another SigOpt trial).

Reply:
You can run sigopt without creating a new experiment. The handling of `create_sigopt` is solely for creating an experiment, as you noticed. But to run an experiment, you simply have to make sure you're using the `SigoptSurpervisedTrainable`. When calling via `run_with_raytune.run`, this equates to ensuring you set `ray_trainable=SigoptSurpervisedTrainable` or simply defining `sigopt_config` in your config which is used to interpret the proper default trainable.

Reply:
OK. I have to say this is quite confusing. I looked through the code and couldn't figure it out. It's really confusing looking through all the parsers, and trying to figure out what's going on with the run command. It used to be so simple before!! It looks like you simply have to "know this" somehow.